### PR TITLE
4.6.0 Rendering Changes article

### PR DESCRIPTION
**Pull request by svdimitr**
Related to: https://github.com/telerik/blazor/issues/6345

### CODE REVIEW

**Comment by svdimitr** [comment]
We must review and merge this PR first https://github.com/telerik/blazor-docs/pull/1664 so that the checks here can pass.

File: upgrade/rendering-changes/4-6-0.md [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+            <th>UI for Blazor 4.5.0</th>
+            <th>UI for Blazor 4.6.0</th>
+        </tr>
+    </thead>
+    <tbody>
+        <tr>
+            <td style="vertical-align:top">
+<pre><code>
+&lt;span class="k-icon k-i-home"&gt;&lt;/span&gt;
+&lt;span class="k-svg-icon k-svg-i-home"&gt;&lt;/span&gt;
```
Comment (dimodi) [inline upgrade/rendering-changes/4-6-0.md:38]: Ideally, the before-after snippets should have the same number of lines. The SVG span doesn't follow this best practice.

**Review by dimodi — APPROVED** [review]

File: upgrade/rendering-changes/4-6-0.md [hunk 2]
```diff
@@ -0,0 +1,71 @@
+---
+title: 4.6.0
+description: Handle the rendering changes in the 4.6.0 release of the Telerik UI for Blazor components.
+page_title: Rendering Changes in the 4.6.0 Release
+slug: changes-in-4-6-0
+position: 1
+---
+
+# Rendering Changes in 4.6.0
```
[the remark this answers is not in the document]
Comment (svdimitr) [reply upgrade/rendering-changes/4-6-0.md:9]: I wanted a "special" sort - Icons first, and then alphabetically, but swapped the  Filter and Form.  I have fixed that.

File: upgrade/rendering-changes/4-6-0.md (hunk 1 above)
Comment (svdimitr) [reply upgrade/rendering-changes/4-6-0.md:38]: fixed

**Pull request merged**